### PR TITLE
JW-1098: Refactor OID

### PR DESCRIPTION
**Pull request by jablonskim**

### CODE REVIEW

File: windows/vrouter_mod.c [hunk 1]
```diff
@@ -824,609 +719,245 @@ FilterSendNetBufferListsComplete(
...
-            }
+    RtlZeroMemory(oidRequest, sizeof(*oidRequest));
+    VrStoreOidRequestStatusHandle(oidRequest, oidRequestStatus);
 
-            break;
+    oidRequest->Header.Type = NDIS_OBJECT_TYPE_OID_REQUEST;
+    oidRequest->Header.Revision = NDIS_OID_REQUEST_REVISION_1;
+    oidRequest->Header.Size = sizeof(NDIS_OID_REQUEST);
+    oidRequest->RequestType = NdisRequestQueryInformation;
+    oidRequest->Timeout = 0;
```
Comment (m-kostrzewa) [inline windows/vrouter_mod.c]: I think we should specify some timeout here, just in case

File: windows/vrouter_mod.c [hunk 2]
```diff
@@ -824,609 +719,245 @@ FilterSendNetBufferListsComplete(
...
-    NDIS_HANDLE FilterModuleContext,
-    PVOID RequestId
-    )
+FilterCancelOidRequest(NDIS_HANDLE FilterModuleContext, PVOID RequestId)
 {
-    UNREFERENCED_PARAMETER(FilterModuleContext);
-    UNREFERENCED_PARAMETER(RequestId);
+    PSWITCH_OBJECT switchObject = (PSWITCH_OBJECT)FilterModuleContext;
+
+    NdisFCancelOidRequest(switchObject->NdisFilterHandle, RequestId);
```
Comment (m-kostrzewa) [inline windows/vrouter_mod.c]: They don't call `NdisFCancelOidRequest` anywhere so why do we call it?

File: windows/vrouter_mod.c (hunk 1 above)
Comment (jablonskim) [reply windows/vrouter_mod.c]: There was no timeout in the original SxBase code

File: windows/vrouter_mod.c (hunk 2 above)
Comment (jablonskim) [reply windows/vrouter_mod.c]: According to documentation we should pass cancel request to lower layer driver.

**Pull request merged**